dev/com.ibm.ws.logging_2_fat/fat/src/com/ibm/ws/logging/fat/LogThrottleTest.java

```diff
@@ -119,8 +133,9 @@ public void testLogThrottlingActiveLowOccurrence() throws Exception {
     @Test
     public void testLogThrottlingActiveHighOccurrence() throws Exception {
         setUp(defaultServer, "testLogThrottlingActiveHighOccurrence");
-        spamServerConfigurationUpdates(25);
-        List<String> lines = serverInUse.findStringsInLogs("CWWKG0016I");
+        hitWebPage("logger-servlet", "LoggerServlet", false, "numMessages=25");
+
+        List<String> lines = serverInUse.findStringsInLogs("TESTA0001W");
         assertEquals("Configuration updated message wasn't printed the correct number of times.", lines.size(), 5);
```

Comment:
Should the assertEquals string changed to "Test message TESTA0001W wasn't printed the correct number of times"?